### PR TITLE
Added License Scoring Part

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

File: f8a_worker/workers/stackaggregator_v2.py [hunk 1]
```diff
@@ -86,21 +91,36 @@ def extract_component_details(component):
         "osio_user_count": component.get("osio_user_count", -1),
         "latest_version": latest_version,
         "github": github_details,
-        "code_metrics": code_metrics
+        "code_metrics": code_metrics,
     }
-    return component_summary, licenses
+    return component_summary, licenses, license_dict
```
Comment (jpopelka) [inline f8a_worker/workers/stackaggregator_v2.py]: all members of this 3-tuple contain `licenses`, isn't that redundant ?

File: f8a_worker/workers/stackaggregator_v2.py [hunk 2]
```diff
@@ -112,7 +132,8 @@ def aggregate_stack_data(stack, manifest_file, ecosystem, deps):
                 "recommendation_ready": True, #based on the percentage of dependecies analysed
                 "total_licenses": len(stack_distinct_licenses),
                 "distinct_licenses": list(stack_distinct_licenses),
-                "stack_license_conflict": False,
+                "stack_license_conflict": True if stack_license[0] is None else False,
```
Comment (jpopelka) [inline f8a_worker/workers/stackaggregator_v2.py]: or simply
`"stack_license_conflict": stack_license[0] is None`,

Comment (miteshvp) [inline f8a_worker/workers/stackaggregator_v2.py]: Yup, but just for better readability.

File: f8a_worker/workers/stackaggregator_v2.py (hunk 1 above)
Comment (miteshvp) [inline f8a_worker/workers/stackaggregator_v2.py]: That's a very good catch. I was just modifying on top of other code, but let me rework on this.

**Comment by miteshvp** [comment]
@jpopelka can we merge this if you're ok?

File: f8a_worker/workers/stackaggregator_v2.py [hunk 3]
```diff
@@ -86,21 +84,42 @@ def extract_component_details(component):
...
+                'license': component_data['licenses']
+            }
             dependencies.append(component_data)
-            licenses.extend(component_licenses)
+            licenses.extend(component_data['licenses'])
+            license_score_list.append(license_scoring_input)
 
     stack_distinct_licenses = set(licenses)
+    # Call License Scoring to Get Stack License
+    license_scoring_url = "http://stack-license-stack-license.dev.rdu2c.fabric8.io/api/v1/stack_license"
```
Comment (msrb) [inline f8a_worker/workers/stackaggregator_v2.py]: What's this?

Comment (miteshvp) [inline f8a_worker/workers/stackaggregator_v2.py]: This is the hard-coded url for license-scoring. @rouseguy will have to take it to the next level of setting env variables and stuff.

**Comment by miteshvp** [comment]
Ok, will have new commits to handle URL params by way of openshift env and also will add a more performant worker in terms of osio-usage-count.

**Pull request closed without merging**